src/gui/previewlistdelegate.h

```diff
@@ -24,73 +24,75 @@
  * modify file(s), you may extend this exception to your version of the file(s),
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
- *
- * Contact : chris@qbittorrent.org
  */
 
 #ifndef PREVIEWLISTDELEGATE_H
 #define PREVIEWLISTDELEGATE_H
 
+#include <QApplication>
 #include <QItemDelegate>
-#include <QStyleOptionProgressBar>
-#include <QStyleOptionViewItem>
 #include <QModelIndex>
 #include <QPainter>
-#include <QApplication>
-#include "base/utils/misc.h"
-#include "base/utils/string.h"
-#include "previewselect.h"
-
+#include <QStyleOptionProgressBar>
+#include <QStyleOptionViewItem>
 #ifdef Q_OS_WIN
 #include <QProxyStyle>
 #endif
 
-class PreviewListDelegate: public QItemDelegate {
-  Q_OBJECT
+#include "base/utils/misc.h"
+#include "base/utils/string.h"
+#include "previewselect.h"
+
+class PreviewListDelegate : public QItemDelegate
+{
+    Q_OBJECT
 
-  public:
-    PreviewListDelegate(QObject *parent=0) : QItemDelegate(parent) {}
+public:
+    PreviewListDelegate(QObject *parent = nullptr) : QItemDelegate(parent) {}
```

Comment:
Coding Style forbids constructor with initializer list is placed in one line.

Reply:
Oops missed that one.